### PR TITLE
K8SPXC-527 Add files for redhat certification

### PR DESCRIPTION
**Pull request by cap1984**
[K8SPXC-527](https://jira.percona.com/browse/K8SPXC-527)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| smart-update | failed |
| recreate | passed |
| big-data | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| operator-self-healing | failed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | failed |
| big-data | passed |
| scaling | passed |
| smart-update | failed |
| recreate | passed |
| haproxy | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| upgrade | passed |
| limits | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |

File: deploy/csv/redhat/1.4.0/manifests/perconaxtradbclusters.pxc.percona.com.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    listKind: PerconaXtraDBClusterList
+    plural: perconaxtradbclusters
+    singular: perconaxtradbcluster
+    shortNames:
+      - pxc
+      - pxcs
+  scope: Namespaced
+  version: v1-4-0
+  versions:
+    - name: v1-4-0
```
Comment (tplavcic) [inline deploy/csv/redhat/1.4.0/manifests/perconaxtradbclusters.pxc.percona.com.crd.yaml:18]: versions seem unordered, which is hard to maintain, same goes for other files

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: deploy/csv/redhat/1.4.0/manifests/perconaxtradbclusters.pxc.percona.com.crd.yaml (hunk 1 above)
Comment (cap1984) [reply deploy/csv/redhat/1.4.0/manifests/perconaxtradbclusters.pxc.percona.com.crd.yaml:18]: done

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| smart-update | failed |
| scaling | passed |
| storage | passed |
| recreate | passed |
| big-data | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| operator-self-healing | failed |
| affinity | passed |
| security-context | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scheduled-backup | failed |
| users | failed |

**Pull request merged**